backend/src/graphql_api/stable_coin.rs

```diff
@@ -425,4 +434,57 @@ impl QueryStableCoins {
         );
         txn_summary
     }
+
+    async fn live_stablecoins(&self, _ctx: &Context<'_>) -> anyhow::Result<Vec<StableCoin>> {
+        let endpoint_env = env::var("CCDSCAN_INDEXER_GRPC_ENDPOINTS");
+        if endpoint_env.is_err() {
+            return Err(anyhow::anyhow!(
+                "Stablecoin data is not available. Only available in Devnet"
+            ));
+        }
+        let app = App::parse();
+        let mut client = v2::Client::new(app.endpoint).await.context("Failed to create client")?;
+
+        let mut response = client
+            .get_token_list(&BlockIdentifier::LastFinal)
+            .await
+            .context("Failed to get token list")
+            .unwrap();
+        let mut coins: Vec<StableCoin> = vec![];
+
+        while let Some(token_id) = response.response.next().await.transpose().unwrap() {
+            let token_info =
+                client.get_token_info(token_id.clone(), BlockIdentifier::LastFinal).await.unwrap();
```

Comment:
Don't unwrap, use `.context("<insert-proper-error>")` instead if we are still going with `anyhow`,
Similarly for every unwrap below